calculator.js

```diff
@@ -1,5 +1,53 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+
+function number_validation(number) {
+  if (isNaN(number)) {
+    return false;
+  }
+  else if (number == null || number === ' ') {
+    return false;
+  }
+  else {
+    return Number(number)
+  }
 }
```

Comment:
I'd recommend `throw`ing exceptions here to signal an error instead of returning `false`.

Also, it's generally preferred to put the `else` on the same line as the closing curly brace for the `if`:

```suggestion
function number_validation(number) {
  if (isNaN(number)) {
    throw new TypeError("Number can't be NaN!");
  } else if (number == null || number === ' ') {
    throw new TypeError("Number can't be empty!");
  } else {
    return Number(number)
  }
}
```